### PR TITLE
Downgrade Merchant to Registered User

### LINKED ISSUE #27: User Story 44, Admin can downgrade a merchant to regular user
```
As an admin user
When I visit a merchant's dashboard ("/admin/merchants/6")
I see a link to "downgrade" the merchant's account to become a regular user
The merchant themselves do NOT see this link
When I click on that link
I am redirected to ("/admin/users/6") because the merchant is now a regular user
And I see a flash message indicating the user has been downgraded
The next time this user logs in they are no longer a merchant
All items for sale by this user are disabled
Only admins can see the "downgrade" button
Only admins can reach any route necessary to downgrade the merchant to user status
```

### Checklist:

- [ ] base tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written (if required)
- [ ] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #33

child of #79

### PR DESCRIPTION
**Pull request by csvlewis**
- Added downgrade route and action for admin merchants controller
- Added link to downgrade merchant on admin merchants show page
- Added functionality to disable a merchant's items when they are downgraded
- User Story 44 complete, closes #27

Here's the functionality for downgrading merchants to regular users. It's almost exactly the same as the upgrade function for users, but it also disables all of the merchant's items at the same time. I disabled the items by iterating through the merchant.items and updating their status with Ruby, but if you think there's a better way to do this let me know.

### CODE REVIEW

**Review by allieIsOkay — APPROVED** [review]

File: app/controllers/admin/merchants_controller.rb [hunk 1]
```diff
@@ -4,4 +4,14 @@ def show
     @orders = Order.pending_orders(@merchant)
     redirect_to admin_user_path(@merchant) if @merchant.registered?
   end
+
+  def downgrade
```
Comment (teresa-m-knowles) [inline app/controllers/admin/merchants_controller.rb:8]: This is a good workaround for now, I'm fine with it. Great job

**Review by teresa-m-knowles — APPROVED** [review]

**Review by timnallen — APPROVED** [review]
Nice work, Chris!

File: app/controllers/admin/merchants_controller.rb (hunk 1 above)
Comment (allieIsOkay) [reply app/controllers/admin/merchants_controller.rb:8]: We might be able to just call update on the merchant's items directly without doing the iteration, but I'm more than okay with this for now.
